invenio_swh/tasks.py

```diff
@@ -52,8 +53,9 @@ def process_published_record(pid):
             service.complete(deposit.id, uow=uow)
             uow.commit()
     except Exception as exc:
+        # Don't retry the task if failed.
         current_app.logger.exception("Failed to complete deposit archival.")
-        raise
+        return
```

Comment:
re-raising would retry this task, and the task would try to create the deposit again. We don't want that